src/components/Form.js

```diff
@@ -1,29 +1,53 @@
-import React from 'react';
-
-const Form = () => (
-  <section className="form">
-    <h3> ADD NEW BOOK </h3>
-    {' '}
-    <form>
-      <input type="text" placeholder="Book title" required />
-      <select>
-        <option value="action"> Action </option>
-        {' '}
-        <option value="comedy"> Comedy </option>
-        {' '}
-        <option value="horror"> Horror </option>
-        {' '}
-        <option value="fiction"> Fiction </option>
-        {' '}
-        <option value="classics"> Classics </option>
-        {' '}
-      </select>
-      {' '}
-      <button type="submit"> ADD BOOK </button>
-      {' '}
-    </form>
-    {' '}
-  </section>
-);
+import React, { useState } from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+import { addBook } from '../redux/books/books';
+
+const Form = () => {
+  const [title, setTitle] = useState('');
+  const [category, setCategory] = useState('');
+  const dispatch = useDispatch();
+  const data = useSelector((state) => state.books);
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const newBook = {
+      id: `item_${data.length + 1}`,
+      title,
+      category,
+    };
+    dispatch(addBook(newBook));
+    setTitle('');
+    setCategory('');
+  };
```

Comment:
- [ ] according to the requirements you should set the title and author for the book, the Category is not a requirement but the author is.